core/src/main/resources/hudson/security/csrf/CrumbFilter/retry.properties

```diff
@@ -0,0 +1,4 @@
+blurb = The URL you're trying to access requires that requests be sent using POST (like a form submission). \
+  The button below allows you to retry accessing this URL using POST. \
+  URL being accessed:
+warning = If you were sent here from an untrusted source, please proceed with caution.
```

Comment:
not sure why, but GitHub UI does not recognize it as a new variable. Maybe `:` needs to be escaped

Reply:
I am guessing Linguist is not very good at parsing line escapes in properties files.